src/winmain.cpp

```diff
@@ -1311,8 +1352,21 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE, PWSTR lpszCmdLine, int)
 
 		// Ask user if he/she want to do update
 		wstring updateAvailable = nativeLang.getMessageString("MSGID_UPDATEAVAILABLE");
-		if (updateAvailable == L"")
+		if (updateAvailable.empty())
 			updateAvailable = MSGID_UPDATEAVAILABLE;
+
+		wstring versionCurrent = nativeLang.getMessageString("MSGID_VERSIONCURRENT");
+		if (versionCurrent.empty())
+			versionCurrent = MSGID_VERSIONCURRENT;
+		versionCurrent += L" " + productVersionToFileVersion(version);
+
+		wstring versionNew = nativeLang.getMessageString("MSGID_VERSIONNEW");
+		if (versionNew.empty())
+			versionNew = MSGID_VERSIONNEW;
+		versionNew += L" " + gupDlInfo.getVersion();
+
+		updateAvailable += L"\n\n" + versionCurrent;

```

Comment:
Npp `gup.xml` is set to `<MessageBoxTitle isModal="no">`. Hence this change is not applied if `isModel` is set to `yes`. However, that modal dialog is not updated for long I guess, as I could see the translation is also not updated. 
Pls see here (line 1325)- 
https://github.com/notepad-plus-plus/wingup/blob/50021b7411ea3204bd2564cbfc168601d6870b09/src/winmain.cpp#L1320-L1327

Reply:
Thank you for your heads up.
I'll make it work.